.github/workflows/delete_storybook.yml

```diff
@@ -12,12 +12,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    # Setup gcloud CLI
-    - uses: google-github-actions/setup-gcloud@v0
+    - id: 'auth'
```

Comment:
I would like to test if updating auth works ok, because setting up gcloud was failing and not deleting images